### ISSUE TITLE
Missing amcl/HypothesisSet.h in building amcl

### ISSUE THREAD

**Issue by cinchcircuits** [mayfieldrobotics/navigation#35]
When attempting to build a deb of amcl, I get the following:

[ 80%] Building CXX object CMakeFiles/amcl.dir/src/amcl_node.cpp.o
/usr/bin/c++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"amcl\" -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -DNDEBUG -D_FORTIFY_SOURCE=2  -I/root/embedded-ansible/navigation/amcl/obj-arm-linux-gnueabihf/devel/include -I/root/embedded-ansible/navigation/amcl/include/amcl -I/root/embedded-ansible/navigation/amcl/include/amcl/map -I/root/embedded-ansible/navigation/amcl/include/amcl/sensors -I/root/embedded-ansible/navigation/amcl/include/amcl/pf -I/opt/ros/indigo/include -I/usr/include/eigen3    -o CMakeFiles/amcl.dir/src/amcl_node.cpp.o -c /root/embedded-ansible/navigation/amcl/src/amcl_node.cpp
/root/embedded-ansible/navigation/amcl/src/amcl_node.cpp:48:32: fatal error: amcl/HypothesisSet.h: No such file or directory
 #include "amcl/HypothesisSet.h"
                                ^

Sure enough, the file does not exist. Is this include needed?

**Comment by KaijenHsiao** [comment]
yup, that include is needed.  It's a new message type used by @Boberito25 in global localization.  I forgot to check his pull request for message exporting in the CMakelists.  @Boberito25, can you add installation lines for your new message?

**Comment by Boberito25** [comment]
https://github.com/mayfieldrobotics/navigation/pull/37

**Comment by KaijenHsiao** [comment]
Aaaaactually, I'm guessing this was fixed by https://github.com/mayfieldrobotics/navigation/pull/38 and https://github.com/mayfieldrobotics/navigation/pull/37 is not actually needed.  @cinchcircuits can you check if this is fixed now?
